src/main.js

```diff
@@ -161,6 +161,7 @@ function getRandomIndex(array) {
 function goBackToMain() {
   mainPosterView.classList.remove("hidden");
   savedPostersView.classList.add("hidden");
+  posterFormView.classList.add("hidden");
```

Comment:
Bingo! Yeah, definitely want to hide that _Make Your Own Poster Section_ on the `Main` page.